client/src/components/Headings.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+function Headings() {
+  return (
+    <div>
+      <h1>welcome</h1> 
+
+
```

Comment:
Could we please remove the empty lines here?

Also I think the component could be renamed to Heading.js (because it returns a singular heading)